.github/workflows/main.yml

```diff
@@ -0,0 +1,42 @@
+on: ["pull_request"]
+name: Proxy repository
```

Comment:
Lets name this workflow as `Static checks` rather than `Proxy repository`

Reply:
+1 on this